vault/core.go

```diff
@@ -1703,6 +1709,11 @@ func (c *Core) postUnseal() (retErr error) {
 	}
 	c.metricsCh = make(chan struct{})
 	go c.emitMetrics(c.metricsCh)
+
+	for _, v := range c.postUnsealFuncs {
```

Comment:
Can we add a comment here that this should be the last block to be executed in this function?

Reply:
Done.